### ISSUE TITLE
Scroll Section to top when active trigger in ScrollMagic

### ISSUE THREAD

**Issue by thallysondias** [janpaepke/ScrollMagic#234]
Hello Jan,

I try use Scrolltomagic to scroll section to top when trigger is activated.
http://jsfiddle.net/thallysondias/14ktjsb7/4/

My objetive is +- like this:
http://jsfiddle.net/puleos/j67eG/40/

When section is activated, the section go to top.
How can make the section go to top when it is activated 
i try use: 
```
.setTween(TweenMax.to(divfocus[1], 2, {scrollTo:{y:50}, ease:Back.easeOut}));
```
but don't works

**Comment by krnlde** [comment]
Maybe you mean "pinning"? Review the following links and give feedback if those fit your needs:

* http://janpaepke.github.io/ScrollMagic/examples/basic/simple_pinning.html
* http://janpaepke.github.io/ScrollMagic/examples/expert/cascading_pins.html
* http://codepen.io/grayghostvisuals/pen/vlodG
* http://codepen.io/grayghostvisuals/pen/enxAp

**Comment by janpaepke** [comment]
Hi Thally I think I understand what you are trying to achieve but this is __not__ what ScrollMagic is intended for.
Forcing certain break positions by scrolling programmatically to them falls in the category of scroll-jacking. You're  basically taking over the scrolling for the user.
ScrollMagic is a scroll interaction plugin. It let's you react to the current scroll position and is not meant to be used to manipulate that position.

I personally consider it very bad UX design to do scroll jacking, while keeping the scroll bar. The reason is there will be problem like jitters, if the user doesn't scroll using a trackpad or mouse wheel, but the scrollbar by dragging it down. This happens due to conflicting input from the programmatic and user scroll. This phenomenon can be easily observed in your second example.

For that same reason both plugins I would recommend for full page scrolling ([OnePageScroll](http://www.thepetedesign.com/demos/onepage_scroll_demo.html) and [fullPage.js](http://alvarotrigo.com/fullPage/)) do not show a scrollbar.
Users can only scroll with a mouse wheel or using navigation buttons.

Bottom line: Either leave it up the user where he wants to stop scrolling or hide the scrollbar and use one of the above mentioned plug-ins.

To learn more about how to implement ScrollMagic with these plugins, check here: https://github.com/janpaepke/ScrollMagic/issues/220#issuecomment-70084198

regards,
J

**Comment by thallysondias** [comment]
Thanks very much for aswer
But. @krnlde suggest a best solution for my case:
http://codepen.io/grayghostvisuals/pen/vlodG

Thanks again

**Comment by alvarotrigo** [comment]
> The reason is there will be problems like jitters and jerks, if the user doesn't scroll using a trackpad or mouse wheel, but using the scrollbar by dragging it down.

Not if implemented correctly.

> For that same reason both plugins I would recommend for full page scrolling (OnePageScroll and fullPage.js) do not show a scrollbar.
> Users can only scroll with a mouse wheel or using navigation buttons.

Not true :)
fullPage.js provides the option `scrollBar:true` and it can be used in combination with scroll bar as you can see in [this example](http://alvarotrigo.com/fullPage/examples/scrollBar.html).

**Comment by krnlde** [comment]
Could you please provide a correctly implemented example, so everyone can profit?

**Comment by alvarotrigo** [comment]
@krnlde not sure who are you asking to, but I already provided an online example.
